tuist/tuist
# Add option to export raw xcodebuild logs to a given file

### Short description 📝
When running acceptance tests via `xcodebuild > XCTest`, logs are not forwarded through `xcodebuild`'s standard output. This motivated me to add a new option to commands, `--raw-xcodebuild-logs-path`, for Tuist to output the raw logs to a given path. In a follow-up PR I plan to use that flag from acceptance tests to output the logs as artifacts of the GitHub Actions workflows. This will improve the experience of debugging failing acceptance tests.

### How to test the changes locally 🧐
You can run `build` against a project passing that option. A file should get created and the raw logs should be dumped there.

### Contributor checklist ✅


### Reviewer checklist ✅
